### PR TITLE
Patch files

### PR DESCRIPTION
**Pull request by nlpet**

### CODE REVIEW

File: src/testModify.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
-const testUser = "tester";
+const testUser = "admin";
```
[the remark this answers is not in the document]
Comment (nlpet) [reply src/testModify.js:2]: another comment

File: src/index.js [hunk 2]
```diff
@@ -13,19 +13,6 @@ import App from "./components/App";
 import reducer from "./reducers";
 import {initialState, typeConversions} from "./constants";
 
-const localStorageDeserializer = (state: string) => {
-  if (state) {
-    const jsState = JSON.parse(state);
-    mapObjIndexed((modeState, mode) => {
-      mapObjIndexed((type, variable) => {
-        jsState[mode][variable] = immutable[type](jsState[mode][variable]);
```
Comment (nlpet) [inline src/index.js:21]: comment between lines 21 and 22.

File: tests/actions.tests.js [hunk 3]
```diff
@@ -5,25 +5,6 @@ import test from "ava";
 import * as actions from "../src/actions";
 import * as actionTypes from "../src/constants/actionTypes";
 
-test("generate action to change mode (navigation)", t => {
```
Comment (schottkey7) [inline tests/actions.tests.js]: Test line 8

File: src/utils.js [hunk 4]
```diff
@@ -13,19 +13,6 @@ export const safelyGetHtmlElementAndValue = (
   return [value, element];
 };
 
-export const numToLetter = fromJS({
```
Comment (nlpet) [inline src/utils.js:16]: 4th line of first block

File: src/utils.js [hunk 5]
```diff
@@ -212,19 +199,7 @@ export function setDifficulty(difficulty: string, numberOfDigits: number) {
   return Math.ceil(numberOfDigits * multipliers[difficulty]);
 }
 
-export function getEmoji(p: number) {
```
Comment (nlpet) [inline src/utils.js]: 4th line of second block

File: src/containers/Games.js [hunk 6]
```diff
@@ -29,7 +29,7 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = dispatch => ({
-  actions: bindActionCreators(actions, dispatch),
```
Comment (nlpet) [inline src/containers/Games.js:32]: Some random comment (part of a review)

File: src/index.js [hunk 7]
```diff
@@ -13,19 +13,6 @@ import App from "./components/App";
 import reducer from "./reducers";
 import {initialState, typeConversions} from "./constants";
```
Comment (nlpet) [inline src/index.js:14]: lala (just a single comment, not part of review)

**Review by nora-petrova — CHANGES_REQUESTED** [review]
Needs changes

**Comment by nlpet** [comment]
Test update

**Comment by nlpet** [comment]
Updated comment

**Comment by nlpet** [comment]
More comments

**Comment by nlpet** [comment]
Mooore

**Pull request closed without merging**